### ISSUE TITLE
(open for all) Logo for homepage

### ISSUE THREAD

**Issue by chehak123** [chehak123/GirlMeetup#1]

**Comment by sneha-baser** [comment]
@chehak123  I wanna work on this issue and resolve . could you please assign me.

**Comment by chehak123** [comment]
@sneha-baser It is an open issue so you can share your ideas first then whosoever will be the best would be assigned.

**Comment by anubhab1710** [comment]
Hello, @chehak123 I would like to work on this issue and resolve it under JWoC 2021. I have a fair understanding of the project's aim and wish to create a minimalist logo for the homepage (that can be furthered as a brand). _I request you to please assign this issue to me._

**Comment by tusharjain0022** [comment]
Hello @chehak123, I'm a UI/UX Designer familiar with Figma and AdobeXD and a mentee in JWOC. I have a doubt regarding this issue, whether you want the SVG/landing page intro image to be designed in this issue or the logo/favicon. I can design the logo/favicon for this project.
PS: I would like to request to increase the difficulty label of this issue from `easy` to `medium` or up if possible for whoever resolves this issue under JWOC.

**Comment by chehak123** [comment]
You can work on this issue and share your ideas here. The best chosen will be assigned.

**Comment by chehak123** [comment]
> Hello @chehak123, I'm a UI/UX Designer familiar with Figma and AdobeXD and a mentee in JWOC. I have a doubt regarding this issue, whether you want the SVG/landing page intro image to be designed in this issue or the logo/favicon. I can design the logo/favicon for this project.
> PS: I would like to request to increase the difficulty label of this issue from `easy` to `medium` or up if possible for whoever resolves this issue under JWOC.

We want logo to be designed

**Comment by Swatilekha-Roy** [comment]
I want to contribute to this issue.

**Comment by tusharjain0022** [comment]
I've few designs for the logo/favicon. (Color can be changed later on depending upon the website theme). 
![Screenshot (209)](https://user-images.githubusercontent.com/59157988/105464482-855f1800-5c46-11eb-8162-530c4e48fbf1.png)
You can suggest some changes or can wait for other assignees' designs whichever you find best for you.
I've tried to keep the designs minimal but attractive at the same time.

**Comment by Swatilekha-Roy** [comment]
![20210122_142600_0000](https://user-images.githubusercontent.com/66589183/105469341-14f3dd80-5cbe-11eb-88b7-091751a8a6c3.png)

This is a design for the official logo that I came up with. Any feedback or criticism or any comments as to how to change it will be highly appreciated. Looking forward to your approval, @chehak123 so that I can work on it. Thanks.

**Comment by chehak123** [comment]
@tusharjain0022 I think your 3rd logo is good enough. Just combine meet and up.

**Comment by anubhab1710** [comment]
Hello @chehak123 I have prepared 6 designs that can be used as the official brand logo of GirlMeetup website's homepage, additionally for posting on social media to promote the project. I have kept every logo minimalistic with a basic color scheme to reserve aesthetics and integrity. Requesting your feedback and suggestions for the same. _I will be including the dark mode scheme of the selected logo you prefer, if you assign me to work on this issue._ Thanks a lot.
![All](https://user-images.githubusercontent.com/76936580/105471216-80d74580-5cc0-11eb-8649-64ad899f48a8.jpg)
